pkg/spec/backup.go

```diff
@@ -66,26 +66,15 @@ func (bp *BackupPolicy) Validate() error {
 }
 
 type StorageSource struct {
+	// PV represents a Persistent Volume resource, operator will claim the
+	// required size before creating the etcd cluster for backup purpose.
```

Comment:
~~for backup purpose~~ .

This is not true after we have PV support for etcd pod.

Reply:
That comment is for `StorageSource` that is a backupPolicy only option.

Reply:
ok